src/main/java/com/auction/retailService/controller/CustomerOrderResource.java

```diff
@@ -0,0 +1,129 @@
+package com.auction.retailService.controller;
+
+import java.util.List;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestHeader;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.auction.retailService.constant.ErrorMessageConstant;
+import com.auction.retailService.dto.CustomerOrderDTO;
+import com.auction.retailService.dto.CustomerOrderDetailDTO;
+import com.auction.retailService.dto.OrderPlacedDTO;
+import com.auction.retailService.exception.InvalidDataException;
+import com.auction.retailService.exception.UserNotExistToPerformOperationException;
+import com.auction.retailService.service.CustomerOrderService;
+import com.auction.retailService.service.StoreService;
+import com.auction.retailService.service.UserService;
+
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+
+@Api(value = "CustomerOrder Management API")
+@RestController
+@RequestMapping("/retail/v1")
+public class CustomerOrderResource {
```

Comment:
Since every endpoint mapping you are using starts by "orders" perhaps you could add it to the controller mapping